docs/code/readme_standards.md

```diff
@@ -3,100 +3,135 @@
 
 ## README Template
 
 Please follow the [README template](readme_template.md) *exactly*. This format is
 read/understood by the [Catalog Site](https://material.io/components/web/catalog/)
 
-## Usage
-
-Under the 'Usage' section, you will document all
+## Basic Usage
 
-* HTML Structure
-* CSS Classes
-* JavaScript
-* Sass Variables and Mixins
+The 'Basic Usage' section documents the bare minimum to get the most basic version of this component up and running, including:
+* HTML structure
+* Style import
+* JavaScript import and instantiation, if applicable
 
+~~~
 ### HTML Structure
 
-HTML Structure is documented with a code snippet, e.g.
-
 ```html
 <button class="mdc-button">Foo</button>
 ```
+~~~
 
-Any additional optional HTML should be documented in separate code snippets.
-
-### CSS Classes
+In the 'Styles' subsection, include any mixin invocations necessary to make the component look as desired.
+~~~
+### Styles
 
-CSS Classes are documented in a [tabular format](#tabular-format)
+```scss
+@import "@material/button/mdc-button";
 
+// Include necessary mixin invocations here
 ```
-CSS Class | Description
---- | ---
-`mdc-foo` | Sets the foo
+~~~
+
+In the 'JavaScript Instantiation' subsection, include a note about importing a JavaScript component with a link to our importing documentation.
+~~~
+### JavaScript Instantiation
+
+```js
+import {MDCButton} from '@material/button';
+
+const button = new MDCButton(document.querySelector('.mdc-button'));
 ```
 
-### Sass Variables and Mixins
+>See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+~~~
 
-Sass variables are documented in a tabular format (See Tabular Format section).
+## Variants
 
+If the component has variants, add a subsection for each variant, under the 'Variants' section. Include a short description of the variant along with the necessary HTML structure.
+
+~~~
+### Foo Button
+
+Short description of Foo Button.
+
+```html
+<button class="mdc-button mdc-button--foo">Foo</button>
 ```
-Variable | Description
+~~~
+
+## Style Customization
+
+CSS Classes are documented in a [tabular format](#tabular-format).
+
+~~~
+### CSS Classes
+
+CSS Class | Description
 --- | ---
-`mdc-foo` | Default foo
-```
+`mdc-foo` | Indicates the foo
+~~~
 
-Sass mixins, including their method signature, are documented in a
-[tabular format](#tabular-format).
+Sass mixins are documented in [tabular format](#tabular-format).
+
+~~~
+### Sass Mixins
 
-```
 Mixin | Description
 --- | ---
 `mdc-foo($value)` | Sets the foo
-```
+~~~
 
-### JavaScript
+Add a section for additional information if applicable.
 
-First have a section for the Vanilla Component. The title of the
-section should be the name of the class, formatted as inline code, aka with the \` marks.
-At the top of the section you should reference our generic documentation for
-importing a JavaScript component.
+~~~
+### Additional Information
```

Comment:
How would we handle stuff like the Tips and Tricks, util API, and Caveats sections in Ripple? Would those all end up crammed under here now? Those would get into really tiny subheadings if they had to be nested starting at h4... (See https://github.com/material-components/material-components-web/tree/master/packages/mdc-ripple#tipstricks and onward in the Ripple readme)

Reply:
Those sections can replace 'Additional Information'. Added a note.